config/routes.rb

```diff
@@ -0,0 +1,22 @@
+Rails.application.routes.draw do
+  devise_for :users
+  root 'public_recipes#index'
+  get 'public_recipes', to: 'public_recipes#index'
+  get 'shopping_list', to: 'shopping_list#index', as: 'shopping_list'
+  
+  post 'toggle_public', to: 'recipes#toggle', as: 'toggle_public'
```

Comment:
- [OPTIONAL ] I don't really see the use and importance of this r`ecipes#toggle` router path 🤔  do you think that is useful ? if not please get rid of 🙏🏻 🥺

Reply:
Hi @BenMukebo it is used to toggle the status of a recipe, from private to public. As the requirements indicate that the update action should not be implemented I decide to use this mechanism. Thanks.